demo/core_10_base.js

```diff
@@ -143,32 +169,49 @@ $.physical.moveTo.updateScene_ = function(room) {
 };
 
 $.physical.look = function(cmd) {
-  var html = '<table style="height: 100%; width: 100%;"><tr><td style="padding: 1ex; width: 30%;">';
-  html += '<svg width="100%" height="100%" viewBox="0 0 0 0">' + this.getSvgText() + '</svg>';
-  html += '</td><td>';
-  html += '<h1>' + this.name + $.utils.commandMenu(this.getCommands(user)) + '</h1>';
-  html += '<p>' + this.getDescription() + '</p>';
-  var contents = this.getContents();
-  if (contents.length) {
-    var contentsHtml = [];
-    for (var i = 0; i < contents.length; i++) {
-      contentsHtml[i] = contents[i].name +
-          $.utils.commandMenu(contents[i].getCommands(user));
-    }
-    html += '<p>Contents: ' + contentsHtml.join(', ') + '</p>';
-  }
-  if (this.location) {
-    html += '<p>Location: ' + this.location.name +
-        $.utils.commandMenu(this.location.getCommands(user)) + '</p>';
-  }
-  html += '</td></tr></table>';
+  var html = $.jssp.generateOutput(this.generateLook, this);
   user.writeJson({type: "html", htmlText: html});
 };
 $.physical.look.verb = 'l(ook)?';
 $.physical.look.dobj = 'this';
 $.physical.look.prep = 'none';
 $.physical.look.iobj = 'none';
 
+$.physical.generateLook = function(request, response, out) {
```

Comment:
I'm wondering if we might want a naming convention for JSSP methods.  Perhaps this should be `$.physical.lookJSSP` (or …`lookJssp`, if you prefer)?